kuksa_viss_client/README.md

```diff
@@ -1,12 +1,12 @@
 # Kuksa VISS Client
-![kuksa.val Logo](../doc/pictures/logo.png)
+![kuksa.val Logo](https://raw.githubusercontent.com/eclipse/kuksa.val/0.2.5/doc/pictures/logo.png)
```

Comment:
Why/When is absolute path needed? Is it because the relative paths will not work if you just install Python source package? Why do we not refer to https://github.com/eclipse/kuksa.val/blob/master/doc/pictures/logo.png? Do we know for how long v0.2.5 will remain, having to update version number as part of every release is cumbersome, and can we actually update the reference before the release?

Reply:
Absolute path do not work on Pypi, where the docs are also used compare old version
https://pypi.org/project/kuksa-viss-client/ where links are broken
and test
https://test.pypi.org/project/kuksa-viss-client/0.2.5.post2/

I try to refer to the tag where possible, so when we move stuff around, pages from older pypi version still look ok (and the tag should be there  to stay in GH)

Reply:
> Absolute path do not work on Pypi

I assume you mean relative path?

Do we by the way have any release instructions for Kuksa.VAL, i.e. what to do when creating a new release? Concerning e.g. update of versions number. If so that checklist could mention that this number does not need to be updated for every new release, but it needs to be checked that the link still works.

Reply:
Jupp, and I we do not have release instructions. yet.